Mailer/README.md

```diff
@@ -887,27 +1176,158 @@ new *email* template via API.
 | --- |---| --- | --- |
 | Authorization | Bearer *String* | yes | API token. |
 
+##### *Body:*
+
+```json5
+{
+  // optional
+  "public_listing": true // Boolean; Get only published newsletters
+}
+```
+
 ##### *Example:*
 
 ```shell
-curl -X GET \
+curl -X POST \
```

Comment:
**MAJOR:** This is a breaking change (you're changing GET api to POST). Please maintain backwards compatibility and keep this API defined via `GET`.